src/modules/code.js

```diff
@@ -0,0 +1,146 @@
+import { list, todo } from './variables.js';
+
+let todoList = [
+  {
+    index: 'aa',
+    description: 'to shopping-mall',
+  },
+  {
+    index: 'ab',
+    description: 'reading',
+  },
+  {
+    index: 'ac',
+    description: 'hangout',
+  },
```

Comment:
- [ ] I think it would be better if you remove all the predefined tasks as you can dynamically add new ones from your `Todo-list` form and they are not needed. If you check, when you reload the page all the predefined tasks are recreated for each reloading action. I know you can do it :+1: :+1: